homeassistant/components/qbittorrent/sensor.py

```diff
@@ -4,122 +4,170 @@
+        suggested_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
+        value_fn=lambda coordinator: float(
+            coordinator.data["server_state"]["dl_info_speed"]
+        ),
     ),
     QBittorrentSensorEntityDescription(
         key=SENSOR_TYPE_UPLOAD_SPEED,
-        name="Up Speed",
+        translation_key="upload_speed",
         icon="mdi:cloud-upload",
         device_class=SensorDeviceClass.DATA_RATE,
-        native_unit_of_measurement=UnitOfDataRate.KIBIBYTES_PER_SECOND,
-        state_class=SensorStateClass.MEASUREMENT,
-        value_fn=lambda data: format_speed(data["server_state"]["up_info_speed"]),
+        native_unit_of_measurement=UnitOfDataRate.BYTES_PER_SECOND,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
+        value_fn=lambda coordinator: float(
+            coordinator.data["server_state"]["up_info_speed"]
+        ),
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_ALL_TORRENTS,
+        translation_key="all_torrents",
+        native_unit_of_measurement="torrents",
+        value_fn=lambda coordinator: count_torrents_in_states(coordinator, []),
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_ACTIVE_TORRENTS,
+        translation_key="active_torrents",
+        native_unit_of_measurement="torrents",
+        value_fn=lambda coordinator: count_torrents_in_states(
+            coordinator, ["downloading", "uploading"]
+        ),
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_INACTIVE_TORRENTS,
+        translation_key="inactive_torrents",
+        native_unit_of_measurement="torrents",
+        value_fn=lambda coordinator: count_torrents_in_states(
+            coordinator, ["stalledDL", "stalledUP"]
+        ),
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_PAUSED_TORRENTS,
+        translation_key="paused_torrents",
+        native_unit_of_measurement="torrents",
+        value_fn=lambda coordinator: count_torrents_in_states(
+            coordinator, ["pausedDL", "pausedUP"]
+        ),
     ),
 )
 
 
 async def async_setup_entry(
     hass: HomeAssistant,
     config_entry: ConfigEntry,
-    async_add_entites: AddEntitiesCallback,
+    async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up qBittorrent sensor entries."""
+
     coordinator: QBittorrentDataCoordinator = hass.data[DOMAIN][config_entry.entry_id]
-    entities = [
-        QBittorrentSensor(description, coordinator, config_entry)
+
+    async_add_entities(
+        QBittorrentSensor(coordinator, config_entry, description)
         for description in SENSOR_TYPES
-    ]
-    async_add_entites(entities)
+    )
 
 
 class QBittorrentSensor(CoordinatorEntity[QBittorrentDataCoordinator], SensorEntity):
     """Representation of a qBittorrent sensor."""
 
-    entity_description: QBittorrentSensorEntityDescription
+    _attr_has_entity_name = True
```

Comment:
```suggestion
    _attr_has_entity_name = True
    entity_description: QBittorrentSensorEntityDescription
```

Reply:
Thank you. This is good knowledge to watch out for. Learning my python linting issues